ui/options.go

```diff
@@ -138,11 +138,17 @@ func WithValidateFunc(fn func(string) error) Option {
 }
 
 // WithValidateNotEmpty adds a custom validation function to a prompt that
-// checks that the propted string is not empty.
+// checks that the prompted string is not empty.
 func WithValidateNotEmpty() Option {
 	return WithValidateFunc(NotEmpty())
 }
 
+// WithValidateMinLen adds a custom validation function to a prompt that
+// checks the input string meets the minimum length requirement.
+func WithValidateMinLen(length int) Option {
+	return WithValidateFunc(MinLen(length))
+}
```

Comment:
I would use `WithValidateMinLength` and `MinLength` instead of `MinLen`. If we want to shorten it, we can name it `WithMinLength`.